src/app/user-search/user-search.component.html

```diff
@@ -0,0 +1,15 @@
+<div>
+  <label for="formControlSearch">Add people</label>
+  <input [formControl]="queryField"
```

Comment:
Maybe we should use this:
https://ng-bootstrap.github.io/#/components/typeahead/examples
Nicer and with many features